src/Settings/PrivacyMode/PrivacyModeContext.tsx

```diff
@@ -0,0 +1,73 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import React from 'react'
+import {useMutation, UseMutationOptions, useQuery, useQueryClient} from 'react-query'
+
+import {PrivacyModeStatus} from '../../types'
```

Comment:
`PrivacyModeStatus` -> `PrivacyMode`
and the same for all the others

Reply:
Done

Reply:
also, using react query we dont need a context at all